includes/configuration_managers/class-woocommerce-configuration-manager.php

```diff
@@ -114,12 +114,13 @@ public function stripe_data() {
 		}
 		$stripe      = $gateways['stripe'];
 		$stripe_data = [
-			'enabled'            => 'yes' === $stripe->get_option( 'enabled', false ) ? true : false,
-			'testMode'           => 'yes' === $stripe->get_option( 'testmode', false ) ? true : false,
-			'publishableKey'     => $stripe->get_option( 'publishable_key', '' ),
-			'secretKey'          => $stripe->get_option( 'secret_key', '' ),
-			'testPublishableKey' => $stripe->get_option( 'test_publishable_key', '' ),
-			'testSecretKey'      => $stripe->get_option( 'test_secret_key', '' ),
+			'enabled'             => 'yes' === $stripe->get_option( 'enabled', false ) ? true : false,
+			'testMode'            => 'yes' === $stripe->get_option( 'testmode', false ) ? true : false,
+			'publishableKey'      => $stripe->get_option( 'publishable_key', '' ),
+			'secretKey'           => $stripe->get_option( 'secret_key', '' ),
+			'testPublishableKey'  => $stripe->get_option( 'test_publishable_key', '' ),
+			'testSecretKey'       => $stripe->get_option( 'test_secret_key', '' ),
+			'allow_covering_fees' => get_option( 'newspack_donations_allow_covering_fees', true ),
```

Comment:
Non-blocking: should we care about case consistency here (lower camel vs. snake)?

Reply:
I think we should, but I'd opt for using snake case rather than camel case. Something for a later PR I think.